Makefile

```
@@ -534,6 +534,8 @@ endef
 %:
 	# Check if we have the CMP tool installed
 	cmp $(ROOT_DIR)/Makefile $(ROOT_DIR)/Makefile >/dev/null 2>&1; if [ $$? -gt 0 ]; then printf "$(MSG_NO_CMP)"; exit 1; fi;
+	# Ensure that python3 is installed. This check can be removed after python is used in more places.
+	if ! python3 --version 1> /dev/null 2>&1; then printf "$(MSG_PYTHON_MISSING)"; fi
```

```suggestion
	if ! python3 --version >/dev/null 2>&1; then printf "%s" "$(MSG_PYTHON_MISSING)"; fi
```
`SC2059: Don't use variables in the printf format string. Use printf "..%s.." "$foo".`
Also change the stdout redirection to be consistent with the rest of the file.